cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -155,21 +155,24 @@ endif()
 # leaving it to the default source file language.
 # This logic is handled in the blt_add_library/executable
 # macros
-blt_import_library(NAME          cuda
-                   COMPILE_FLAGS ${_cuda_compile_flags}
-                   INCLUDES      ${CUDA_INCLUDE_DIRS}
-                   LIBRARIES     ${CUDA_LIBRARIES}
-                   LINK_FLAGS    "${CMAKE_CUDA_LINK_FLAGS}"
-                   EXPORTABLE    ${BLT_EXPORT_THIRDPARTY}
-                   )
+if (NOT TARGET blt::cuda)
```

Comment:
`blt::cuda` is not an actual BLT target. We should be namespacing our targets but for historical reasons we havent yet.